src/components/button/bl-button.stories.mdx

```diff
@@ -77,6 +85,13 @@ ${SingleButtonTemplate({size: 'large', ...args})}
 ${SingleButtonTemplate({size: 'medium', ...args})}
 ${SingleButtonTemplate({size: 'small', ...args})}`
 
+export const LoadingStateTemplate = (args) => html`
+<bl-button icon="play" @click="${(event)=>event.currentTarget.toggleAttribute('loading')}" icon="${ifDefined(args.icon)}"></bl-button>
```

Comment:
If we want to keep this interaction in story, then we need to have a play function to be able to take screenshot of this button in loading state in chromatic. Currently it looks like below:

<img width="513" alt="image" src="https://user-images.githubusercontent.com/127687/222670255-0f136c2a-f680-4e69-a3ff-df39939fba69.png">

Another option would be just putting `SingleButtonTemplate` like other examples here.